frontend/src/styles/Login.module.css

```diff
@@ -0,0 +1,111 @@
+.joinOurCommunity {
+    font-family: Work Sans;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 3.2vh;
+    line-height: 3.2vh;
+
+    margin-top: 10vh;
+
+    text-align: center;
+    letter-spacing: 0.1em;
+
+    color: #000000;
```

Comment:
i usually like to make these be `color: black` so it's more readable than the hex code

same for `#FFFFFFF`, i like to use `color: white` instead